requirements-unit.in

```diff
@@ -1,6 +1,22 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+# Please note this file introduces dependencies from the charm's requirements.in,
+# special attention must be taken when updating this or the other .in file to try
+# to avoid incompatibilities.
+# Rules for editing this file:
+#   * Removing a dependency that is no longer used in the unit test file(s)
+#     is allowed, and should not represent any risk.
+#   * Adding a dependency in this file means the dependency is directly used
+#     in the unit test files(s).
+#   * ALL python packages/libs used directly in the unit test file(s) must be
+#     listed here even if requirements.in is already adding them. This will
+#     add clarity to the dependency list.
+#   * Pinning a version of a python package/lib shared with requirements.in
+#     must not introduce any incompatibilities.
 bcrypt
```

Comment:
Why is that needed?

Reply:
I'm not entirely sure about this one, I did not bother changing the existing deps.

Reply:
Opened an issue to track this: https://github.com/canonical/kubeflow-volumes-operator/issues/99